pkg/producer/convertor/test/flowtype_test.go

```diff
@@ -30,7 +30,7 @@ import (
 
 var (
 	// Hard coding the kafka msg in bytes. Need to figure out a way to generate this.
-	msg1ForFlowType1 = []byte{0x0, 0x0, 0x0, 0xa7, 0xa, 0xa4, 0x1, 0x10, 0x1, 0x18,
+	msg1ForFlowType1 = []byte{0x0, 0x0, 0x0, 0xa4, 0x10, 0x1, 0x18,
```

Comment:
why are the changes in message information here? is it because we don't need to init flowMessage struct?

Reply:
Yes there is no wrapper using oneof.. we just cast all protos to `Protoreflect.Message` format.